vignettes/DataAnalysis.Rmd

```diff
@@ -164,41 +170,44 @@ The default flagging function for the normal approximation method is `Flag_Norma
 The data frame `dfSummary` is derived from `dfFlagged` using the `Summarize()` function. The resulting `dfSummary` data frame will contain the most relevant columns from `dfFlagged` with data sorted in a meaningful way to provide a concise overview of the assessment. Flagged sites will sort earlier than non-flagged sites, with the more "extreme" outliers displayed first. The columns in `dfSummary` include:
 
 -   `GroupID`: Group Identifier (default is Site ID)
+-   `GroupLevel`: Type of Group specified in `GroupID` (Country, Site, QTL)
 -   `Numerator`: Cumulative Number of Event(s) of Interest (in this example, number of AEs reported across subjects); Carried from `dfAnalyzed`
 -   `Denominator`: Cumulative Time on Treatment (in days, across subjects); Carried from `dfAnalyzed`
 -   `Metric`: Rate of Event(s) of Interest (in this example, number of AEs reported over the course of treatment in days)
 -   `Score`: Calculated Residual (per site)
 -   `Flag`: Flag Indicating Possible Statistical Outliers; Valid values for this variable include -2, -1, 0, 1, and 2, which determine the "extremeness" of the outlier. -2 and 2 represent more extreme outliers, -1 and 1 represent less extreme outliers, and 0 represents a non-outlier.
 
-```{r}
-# dfSummary <- Summarize(dfFlagged)
+```{r include = TRUE}
+dfSummary <- Summarize(dfFlagged)
 ```
 
 ```{r, echo = FALSE}
-# dt(dfSummary)
+dt(dfSummary)
```

Comment:
I don't think there's anything to do about this, but it's unfortunate that `GroupID` and `Numerator` happen to both be `83` for the first row of this data. Maybe mention that (that it's a coincidence)? I *think* that row will always be first.

Reply:
I'll just remove the first row of the table to avoid confusion here.